Makefile

```diff
@@ -7,24 +7,12 @@ help:
 
 .PHONY: install-tools
 install-tools: ## Install staticcheck.
-ifeq ($(shell command -v staticcheck 2> /dev/null),)
```

Comment:
If the command exists, it will be skipped, but the command is not necessarily the latest version, so this has been fixed.